backend/app/views/backend/activities/index.html.slim

```diff
@@ -0,0 +1,16 @@
+h2 Activities
```

Comment:
I mentioned in a previews PR review that I would like to not use slim as the template engine.